src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -1336,35 +1336,13 @@ public enum ResolutionPurpose
 
     internal static class AmsiUtils
     {
-        private static string GetProcessHostName(string processName)
-        {
-            return string.Concat("PowerShell_", processName, ".exe_0.0.0.0");
-        }
-
         internal static int Init()
         {
             Diagnostics.Assert(s_amsiContext == IntPtr.Zero, "Init should be called just once");
 
             lock (s_amsiLockObject)
             {
-                Process currentProcess = Process.GetCurrentProcess();
-                string hostname;
-                try
-                {
-                    hostname = string.Concat("PowerShell_", Environment.ProcessPath, "_",
-                        currentProcess.MainModule.FileVersionInfo.ProductVersion);
-                }
-                catch (ComponentModel.Win32Exception)
-                {
-                    // This exception can be thrown during thread impersonation (Access Denied for process module access).
-                    hostname = GetProcessHostName(currentProcess.ProcessName);
-                }
-                catch (FileNotFoundException)
-                {
-                    // This exception can occur if the file is renamed or moved to some other folder
-                    // (This has occurred during Exchange set up).
-                    hostname = GetProcessHostName(currentProcess.ProcessName);
-                }
+                string hostname = string.Concat("PowerShell_", Environment.ProcessPath, "_", PSVersionInfo.ProductVersion);
```

Comment:
I realize you did not change this variable name but `hostname` sends me to the wrong place. Maybe rename this to `appName`?  That is the actual name of the parameter to `AmsiInitialize` api according to https://docs.microsoft.com/en-us/windows/win32/api/amsi/nf-amsi-amsiinitialize

Reply:
Done!